GTG/gtk/ui/preferences.ui

```diff
@@ -0,0 +1,497 @@
+                            <property name="margin-start">10</property>
+                            <child>
+                              <object class="GtkLabel" id="font_title">
+                                <property name="label">Set font type and size</property>
+                                <property name="visible">true</property>
+                                <property name="xalign">0.0</property>
+                                <attributes>
+                                  <attribute name="weight" value="bold"/>
+                                </attributes>
+                              </object>
+                            </child>
+                            <child>
+                              <object class="GtkLabel" id="font_label">
+                                <property name="label">Set the font type and size to be used in task editor</property>
+                                <property name="visible">true</property>
+                                <property name="xalign">0.0</property>
+                                <style>
+                                  <class name="dim-label"/>
+                                </style>
+                                <attributes>
+                                  <attribute name="scale" value="0.833333"/>
+                                </attributes>
+                              </object>
+                            </child>
+                          </object>
+                        </child>
+                      </object>
+                    </child>
+                  </object>
+                </child>
+              </object>
+              <packing>
+                <property name="name">general</property>
+                <property name="title" translatable="yes">General Settings</property>
+                <property name="position">0</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkLabel" id="plugins">
+                <property name="visible">True</property>
+                <property name="label">Plugins. Coming soon.</property>
+                <property name="valign">center</property>              
+              </object>
+              <packing>
+                <property name="name">plugins</property>
+                <property name="title" translatable="yes">Plugins</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkLabel" id="synchronisation">
+                <property name="visible">True</property>
+                <property name="label">Synchronisation Services. Coming soon.</property>
+                <property name="valign">center</property>              
+              </object>
+              <packing>
+                <property name="name">synchronisation</property>
+                <property name="title" translatable="yes">Synchronisation Services</property>
```

Comment:
I think `Synchronisation` should be sufficient and more understandable for users.